hyperjumptech/monika
# Fix: 507 do not send status notif on symon mode

# Monika Pull Request (PR)  
Fix #507 

## What feature/issue does this PR add  
To disable sending periodical status notification when in symon mode

## How did you implement / how did you fix it  
By add filtering to run send status notification function only when there's no symon configuration assigned. 

## How to test  
1. set monika with symon configurations
2. add notification in config
3. run monika
4. make sure there is no status notification sent.
